pandas-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_science/pipeline.py

```diff
@@ -1,34 +1,31 @@
-"""Example code for the nodes in the example pipeline. This code is meant
-just for illustrating basic Kedro features.
-
-Delete this when you start working on your own Kedro project.
+"""
+This is a boilerplate pipeline 'data_science'
+generated using Kedro {{ cookiecutter.kedro_version }}
 """
 
-from kedro.pipeline import node, pipeline
-
-from .nodes import predict, report_accuracy, train_model
-
+from kedro.pipeline import Pipeline, node, pipeline
+from .nodes import split_data, train_model, evaluate_model
 
-def create_pipeline(**kwargs):
+def create_pipeline(**kwargs) -> Pipeline:
     return pipeline(
         [
             node(
-                train_model,
-                ["example_train_x", "example_train_y", "parameters"],
-                "example_model",
-                name="train",
+                func=split_data,
+                inputs=["example_iris_data", "params:model_options"],
+                outputs=["X_train", "X_test", "y_train", "y_test"],
```

Comment:
The old `pandas-iris` is the one that I always refer to as it uses a dictionary of output in a node. With the `data_processing` pipeline being removed, do we have a similar example in other starters?

Reply:
I believe the `astro-airflow-iris` starter also has this example.